InnerEye/ML/configs/histo_configs/classification/DeepSMILEPanda.py

```diff
@@ -28,19 +28,40 @@
     InnerEyeSSLEncoder,
 )
 from InnerEye.ML.configs.histo_configs.classification.BaseMIL import BaseMIL
+from InnerEye.ML.Histopathology.datasets.panda_dataset import PandaDataset
+
+
+local_mode = False
+path_local_data: Optional[Path]
+if local_mode:
+    path_local_data = Path("/tmp/datasets/PANDA_tiles")
+    azure_dataset_id = "Dummy"
+    extra_local_dataset_paths = [Path("/tmp/datasets/PANDA")]
+    extra_azure_dataset_ids = ["Dummy"]
+    num_epochs = 1
+else:
+    path_local_data = None
+    azure_dataset_id = "PANDA_tiles"
+    extra_local_dataset_paths = []
+    extra_azure_dataset_ids = ["PANDA"]
+    num_epochs = 100
```

Comment:
You can bake the local datasets right into the model config, I can explain in a call how that will work out.

Reply:
Thanks, I have now changed this as suggested